README.md

```diff
@@ -13,24 +13,24 @@
 [![devDependency Status](https://img.shields.io/david/dev/jshint/jshint.svg?style=flat)](https://david-dm.org/jshint/jshint#info=devDependencies)
 [![Coverage Status](https://img.shields.io/coveralls/jshint/jshint.svg?style=flat)](https://coveralls.io/r/jshint/jshint?branch=master)
 
-JSHint is a community-driven tool to detect errors and potential problems in
-JavaScript code and to enforce your team's coding conventions. It is very
-flexible so you can easily adjust it to your particular coding guidelines and
-the environment you expect your code to execute in. JSHint is open source and
+JSHint is a community-driven tool that detects errors and potential problems in
+JavaScript code and that enforces your team's coding conventions. Since JSHint 
```

Comment:
We've been moving away from code style conventions, might be better to leave that out